test/e2e/run_test.go

```diff
@@ -48,6 +48,36 @@ var _ = Describe("Podman run", func() {
 		Expect(session).Should(Exit(0))
 	})
 
+	// This test may seem entirely pointless, it is not.  Due to compatibility
+	// and historical reasons, the container name generator uses a globally
+	// scoped RNG, seeded from a global state.  An easy way to check if its
+	// been initialized properly (i.e. pseudo-non-deterministically) is
+	// checking if the name-generator spits out the same name twice.  Because
+	// existing containers are checked when generating names, the test must ensure
+	// the first container is removed before creating a second.
+	It("podman generates different names for successive containers", func() {
+		var names [2]string
+
+		for i := range names {
+			session := podmanTest.Podman([]string{"run", "--cidfile=/dev/stdout", ALPINE, "true"})
```

Comment:
@Luap99 any idea why `--cidfile=/dev/stdout` [isn't working](https://api.cirrus-ci.com/v1/artifact/task/6596184182095872/html/int-podman-debian-12-root-host-boltdb.log.html#t--podman-generates-different-names-for-successive-containers--3) (log)?  It seems to work fine for me locally (F37).

Reply:
Maybe I should use `/proc/self/fd/1` instead?

Reply:
just use `podman create`, faster and will cause less flakes. I don't know why this is not working, the output capturing in ginkgo is a bit special. Interestingly the log shows it is working in the first iteration but then the same command fails in the second iteration.

Reply:
Good idea, thanks Paul.